### ISSUE TITLE
Bad Request when using .addTracksToPlaylist

### ISSUE THREAD

**Issue by cew88** [kaaes/spotify-web-api-android#181]
I'm having issues adding tracks to a playlist when using the .addTracksToPlaylist function. I believe I have all the correct parameters. The method takes in userID, playlistID, and two Map objects (listed as optional in the web documentation, but the Java method requires four input parameters for me). I formatted the Maps with one element. I put "uris" as the key and an array of song URIs as the value.

I noticed that the WebAPI documentation uses a new request URL since the creation of this wrapper in 2016. Spotify says the old URL should work, but their updated blog post was from 2018, so I'm not sure if the old URL still works or not.

I had issues with building this project, so I used the .aar file that someone previously uploaded. Is there a way to edit the URL or perhaps my parameters are incorrect?

Would greatly appreciate any help! Thanks!

**Comment by jiwaCh** [comment]
yes, there is a problem with the wrapper. I ran the code using the Test file which was provided with the wrapper and it yield an error. Did you manage to find a solution or did you end up using another library?
